app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,15 @@
+class RecipesController < ApplicationController
+
+  def show
+    @recipe = Recipe.find(params[:id])
+    @ingredients = @recipe.ingredients
+    @cost = @recipe.ingredient_sum
```

Comment:
It's totally okay to have this @cost and @ingredients instance variables here, but also want to confirm that it's okay to call @recipe.cost or @recipe.ingredients in the view as well. You're not violating MVC by calling a model method. You encapsulating the AR logic involved in the calculation to a model method, but you can call that method from anywhere. Great work!